### PR TITLE
build 1.4.0rc2

### PR DESCRIPTION
**Pull request by akrherz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
*  Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@conda-forge-admin please rerender

### CODE REVIEW

**Comment by akrherz** [comment]
@conda-forge-admin please rerender

**Comment by akrherz** [comment]
ye ole flapping test, will take this once green

**Comment by sgillies** [comment]
Awesome. Thanks @akrherz !

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
     - cython
     - numpy
     - libgdal
```
Comment (weiji14) [inline recipe/meta.yaml:33]: @akrherz, I'm not sure if there's gonna be a 1.4.0rc3 or 1.4.0 will be next, but could you make sure we switch from `libgdal` to `libgdal-core` when the next release happens? Xref https://github.com/conda-forge/rasterio-feedstock/issues/303

Comment (akrherz) [reply recipe/meta.yaml:33]: Yeah, I reviewed that issue today and was trying to make a mental note to make that happen once 1.4.0 comes out!

**Pull request merged**